src/extension/tools/common/toolNames.ts

```diff
@@ -46,6 +47,7 @@ export const enum ToolName {
 	CreateDirectory = 'create_directory',
 	RunVscodeCmd = 'run_vscode_command',
 	GetTaskOutput = 'get_task_output',
+	GetTaskOutputCore = 'get_task_output2',
```

Comment:
I've put the core ones at the bottom of the list with a `Core` prefix, could you match that?